#! /bin/sh /usr/share/dpatch/dpatch-run
## 203_locations.diff        author unknown
##
## DP: This patch changes values in src/slrnfeat.hin

@DPATCH@
diff -Nur slrn-0.9.8.1pl1.orig/src/slrnfeat.h slrn-0.9.8.1pl1/src/slrnfeat.h
--- slrn-0.9.8.1pl1.orig/src/slrnfeat.hin
+++ slrn-0.9.8.1pl1/src/slrnfeat.hin
@@ -70,7 +70,7 @@
  */
 
   /* Root directory names */
-# define SLRN_SPOOL_ROOT	"/var/spool/news"
+# define SLRN_SPOOL_ROOT	"/var/spool/slrnpull/news"
   /* SLRN_NOV_ROOT gives the root directory for overview files
    * if you don't have overview files, leave as SLRN_SPOOL_ROOT for now */
 # define SLRN_SPOOL_NOV_ROOT	SLRN_SPOOL_ROOT
@@ -152,7 +152,9 @@
 #endif				       /* vms */
 
 /* #define OUR_ORGANIZATION "organization-name" */
+#define OUR_ORGANIZATION "/etc/news/organization"
 /* #define OUR_HOSTNAME "host.name.here" */
+#define OUR_HOSTNAME "/etc/mailname"
 
 #if SLRN_HAS_GROUPLENS
 # undef SLRN_USE_SLTCP
@@ -189,7 +191,7 @@
 #define SLRNPULL_NEWS_DIR	"news"
 
 /* File where messages and errors will be placed. */
-#define SLRNPULL_LOGFILE	"log"
+#define SLRNPULL_LOGFILE	"/var/log/news/slrnpull.log"
 
 #if !SLRN_HAS_NNTP_SUPPORT
 # undef SLRN_HAS_GROUPLENS
